CMakeLists.txt

```diff
@@ -282,22 +295,39 @@ if(WITH_SPDK)
   set(HAVE_SPDK TRUE)
 endif(WITH_SPDK)
 
-option(WITH_PMEM "Enable PMEM" OFF)
-if(WITH_PMEM)
-  set(HAVE_PMEM ON)
-  if(NOT WITH_BLUESTORE)
-    message(SEND_ERROR "Please enable WITH_BLUESTORE for using PMEM")
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "i386|i686|amd64|x86_64")
+  # NVML is now PMDK
+  if(WITH_PMEM_DEVICE)
+    option(WITH_PMDK "Enable PMDK libraries" ON)
+  else()
+    option(WITH_PMDK "Enable PMDK libraries" OFF)
+  endif(WITH_PMEM_DEVICE)
+else()
+  # ARM64 PMDK support is still experimental
+  option(WITH_PMDK "Enable PMDK libraries" OFF)
+endif(CMAKE_SYSTEM_PROCESSOR MATCHES "i386|i686|amd64|x86_64")
+if(WITH_PMDK)
+  set(HAVE_PMDK ON)
+  if(NOT WITH_PMEM_DEVICE)
+    message(SEND_ERROR "Please enable WITH_PMEM_DEVICE for using PMDK libraries")
   endif()
+  option(WITH_PMDK_PKG "Use installed PMDK" OFF)
```

Comment:
better off renaming this option to `WITH_SYSTEM_PMDK` to be more consistent.

Reply:
Done